src/backend/routers/agent.py

```diff
@@ -40,6 +40,7 @@ def create_agent(session: DBSessionDep, agent: CreateAgent, request: Request) ->
         tools=agent.tools,
     )
 
+    request.state.agent = agent_data
```

Comment:
What is this for?

Reply:
When creating an agent we need to return it's data, like name, description...